src/pages/en/guides/migration-guide/from-gatsby.md

```diff
@@ -0,0 +1,1199 @@
+
+- **Update your existing Gatsby project dependencies** in `package.json`. Remove any Gatsby-related dependencies and follow all the steps to [install Astro manually](/en/install/manual/) in your existing Gatsby project. Make sure you have [these project files](/en/install/manual/#7-next-steps) and that you have a working home page before continuing. (You may wish to do this on a different branch.)
+
+### Change `static/` to `public/`
+
+1. **Delete** Gatsby's `public` folder. 
+   
+    Gatsby uses the `public` directory for its build output, so you can safely discard this folder. You will no longer need a built version of your Gatsby site. (Astro uses `dist/` by default for the build output.)
+
+2. **Rename** Gatsby's `static` folder to `public`, and use it as Astro's `public/` folder. 
+
+    Gatsby uses a directory named `static/`, whereas Astro uses `public/` for these static assets. You can alternatively copy the contents of `static/` into an existing Astro `public/` folder.
+
+
+### Repurpose config files
+
+Gatsby has several top-level configuration files for configuration options, site and page metadata and generating page routes.
+
+In Astro, many of these features are not handled by separate configuration files, and `astro.config.mjs` is used only for configuring your Astro project and any installed integrations, including SSR adapters. 
+
+The contents of `astro.config.mjs` are not available to other files in your project, so you will write `.astro` components or separate data files (e.g. `.js`, `.json`) for storing site metadata to be used in within your project. 
+
+You have access to every page `<head>` directly, so you can add font sources and stylesheets as needed in individual pages or layout components. Page routing for your blog posts based on GraphQL queries defined in `gatsby-node.js` is replaced with Astro's built-in, automatic file-based routing for files located within `src/pages/`.
+
+Your Astro project will not use any of these `gatsby-*.js` files, but there may be some content that you can reuse:
+
+- `gatsby-config.js`: Move your `siteMetadata: {}` into `src/data/siteMetadata.js` (or `siteMetadata.json`) to import data about your site (title, description, social accounts etc.) into page layouts.
+- `gatsby-browser.js`: Consider adding anything used here directly into your main layout's `<head>` tag.
+- `gatsby-node.js`: This file handles multiple server-side operations that have built-in support in Astro:
+     - GraphQL node customization: You do not need to customize the schema in Astro, but viewing the Gatsby schema may help you with defining types in your Astro projects.
+     - Page creation: Astro supports both [async data loading](/en/reference/api-reference/#getstaticpaths) and [pagination](/en/core-concepts/routing/#pagination) right out of the box. You can cross-reference your `createPages` function to migrate the pages to Astro's methodologies.
```

Comment:
The following suggestion was originally written by @sarah11918:

```suggestion
     - Page creation: Any page file located within `src/pages/` will become a page route in Astro with no `createPages` function needed. However, for dynamic page generation, Astro supports both [async data loading](/en/reference/api-reference/#getstaticpaths) and [pagination](/en/core-concepts/routing/#pagination) right out of the box. You can cross-reference your `createPages` function to migrate to Astro's routing methods.
```

Would you say this is still accurate? I do think it's worth mentioning that you don't necessarily NEED to generate your pages dynamically.. like, at all!